File: two-sum/two-sum.go
func twoSum(nums []int, target int) []int {
    hashTable := make(map[int]int)
    for i, num := range nums{
        if idx, ok := hashTable[target - num]; ok{
            return []int{idx,i}
        }
        hashTable[num] = i
    }
    return nil
} 

File: two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        count = {}
        
        for i, a in enumerate(nums):
            res = target - a
            
            if res in count:
                return [i, count[res]]
            count[a] = i
        return